### PR TITLE
feat(content): New Coalition License description

### PR DESCRIPTION
**Pull request by Arachi-Lover**
**Content (Outfit Description)**

Something that has always bugged me personally in Endless Sky, is the four license images to the left in the following image (Coalition + Wanderer ones), as they always seemed to stand out as not really looking like tangible objects as much as the other licenses.

image

Upon inspecting them recently, I noticed something else, however. All those six licenses in the image, currently, have no description of their own, rather they just reuse the log entry for when you receive each of the respective licenses. The other licenses, such as Successor or Navy licenses, are descriptive, with even a bit of lore on the Pilot's License. It seems like a waste to just have nothing here for these six.

This PR proposes a new description for the Coalition License, giving it a bit of lore now and not being simply a copypaste of the log entry.

## Summary

This PR changes the Coalition License description, as compared in the ingame Outfitter view in the image below, with the current description on the left, and the new one on the right:

image

### CODE REVIEW

File: data/coalition/coalition outfits.txt [hunk 1]
```diff
@@ -430,7 +430,8 @@ outfit "Anti-Materiel Gun"
 outfit "Coalition License"
 	category "Licenses"
 	thumbnail "outfit/coalition license"
-	description "Completed a lot of Coalition jobs and received permission to purchase civilian technology and ships."
+	description "A digital chip in the form of a stylized badge, this document signifies the permission of the owner to browse, employ and deal in transactions involving civilian Coalition technology."
+	description "	There is a history of controversy surrounding this license, as throughout the millennia several Heliarch laws have been in a back and forth regarding the requirements that need to be met for someone to be issued one, starting with the third Civil Goods and Fares Act, just a few decades after the War of Independence, all the way to the relatively recent twenty-sixth addendum to the second article of the Ninth Rectification on Commerce and Traffic, which two hundred years ago made it much more flexible to acquire such a license."
```
Comment (Daeridanii1) [inline data/coalition/coalition outfits.txt:434]: "...made it much more flexible..." reads as being somewhat unclear to me (what is being flexed?) This suggestion also changes the first mention of "requirements" so it doesn't appear twice in the same sentence with the suggested change.
```suggestion
	description "	There is a history of controversy surrounding this license, as throughout the millennia several Heliarch laws have been in a back and forth regarding the conditions that need to be met for someone to be issued one, starting with the third Civil Goods and Fares Act, just a few decades after the War of Independence, all the way to the relatively recent twenty-sixth addendum to the second article of the Ninth Rectification on Commerce and Traffic, which two hundred years ago made the acquisition requirements for such a license much more flexible."
```

Comment (TheGiraffe3) [inline data/coalition/coalition outfits.txt:434]: This last sentence runs on a lot.

**Review by Daeridanii1 — COMMENTED** [review]
A good improvement! It's fun to see more details of the Heliarch bureaucracy.

File: data/coalition/coalition outfits.txt (hunk 1 above)
Comment (Arachi-Lover) [reply data/coalition/coalition outfits.txt]: Any idea on how to break it up?

File: data/coalition/coalition outfits.txt [hunk 2]
```diff
@@ -430,7 +430,8 @@ outfit "Anti-Materiel Gun"
 outfit "Coalition License"
 	category "Licenses"
 	thumbnail "outfit/coalition license"
-	description "Completed a lot of Coalition jobs and received permission to purchase civilian technology and ships."
+	description "A digital chip in the form of a stylized badge, this document signifies the permission of the owner to browse, employ and deal in transactions involving civilian Coalition technology."
+	description "	There is a history of controversy surrounding this license, as throughout the millennia several Heliarch laws have been in a back and forth regarding the conditions that need to be met for someone to be issued one, starting with the third Civil Goods and Fares Act, just a few decades after the War of Independence, all the way to the relatively recent twenty-sixth addendum to the second article of the Ninth Rectification on Commerce and Traffic, which two hundred years ago made the acquisition requirements for such a license much more flexible."
```
Comment (TheGiraffe3) [inline data/coalition/coalition outfits.txt:434]: For some reason GitHub didn't let me suggest changes above.
```suggestion
	description "	There is a history of controversy surrounding this license, as throughout the millennia several Heliarch laws have been in a back and forth regarding the conditions that need to be met for someone to obtain a license. This argument started with the third Civil Goods and Fares Act, created just a few decades after the War of Independence, and has continued all the way to the relatively recent twenty-sixth addendum to the second article of the Ninth Rectification on Commerce and Traffic, which was passed two hundred years ago and made the acquisition requirements for such a license much more flexible."
```
The second part probably still needs a little breakup, but I have no idea where to put it.

File: data/coalition/coalition outfits.txt [hunk 3]
```diff
@@ -430,7 +430,8 @@ outfit "Anti-Materiel Gun"
 outfit "Coalition License"
 	category "Licenses"
 	thumbnail "outfit/coalition license"
-	description "Completed a lot of Coalition jobs and received permission to purchase civilian technology and ships."
+	description "A digital chip in the form of a stylized badge, this document signifies the permission of the owner to browse, employ and deal in transactions involving civilian Coalition technology."
+	description "	There is a history of controversy surrounding this license, as throughout the millennia several Heliarch laws have been in a back and forth regarding the conditions that need to be met for someone to obtain a license. This argument started with the third Civil Goods and Fares Act, created just a few decades after the War of Independence, and has continued all the way to the relatively recent twenty-sixth addendum to the second article of the Ninth Rectification on Commerce and Traffic, which was passed two hundred years ago and made the acquisition requirements for such a license much more flexible."
```
Comment (reticent-rem) [inline data/coalition/coalition outfits.txt:434]: An attempt at breaking up that last sentence:

```suggestion
	description "	There is a history of controversy surrounding this license, as throughout the millennia several Heliarch laws have been in a back and forth regarding the conditions that need to be met for someone to obtain a license. This argument started with the third Civil Goods and Fares Act, created just a few decades after the War of Independence. It has continued all the way to the relatively recent twenty-sixth addendum to the second article of the Ninth Rectification on Commerce and Traffic, passed two hundred years ago, which made the acquisition requirements much more flexible."
```

**Review by TheGiraffe3 — APPROVED** [review]

**Review by tibetiroka — APPROVED** [review]

**Review by roadrunner56 — APPROVED** [review]

**Pull request merged**